frontend/micro-ui/web/micro-ui-internals/packages/modules/sv/src/pages/citizen/Create/SVAcknowledgement.js

```diff
@@ -0,0 +1,89 @@
+import { Banner, Card, CardText, LinkButton, LinkLabel, Loader, Row, StatusTable, SubmitBar } from "@nudmcdgnpm/digit-ui-react-components";
+import React, { useEffect } from "react";
+import { useTranslation } from "react-i18next";
+import { Link, useRouteMatch } from "react-router-dom";
+import { svPayloadData } from "../../../utils";
+
+/* This component, SVAcknowledgement, is responsible for displaying the acknowledgement 
+ of a service request submission. It utilizes the Digit UI library components for 
+ rendering the UI elements. 
+
+ Key functionalities include:
+ 1. **Data Submission**: On component mount (using useEffect), it prepares the data 
+    by extracting the tenantId from the address and formats it using the `svPayloadData` 
+    utility function. It then calls the `useSvCreateApi` hook to submit this data.
+ 2. **Loading State**: While the data submission is in progress, a Loader component 
+    is displayed to indicate that the request is being processed.
+ 3. **Acknowledgement Banner**: Once the data submission is complete, a Banner 
+    component is rendered to show the result of the submission, which can either 
+    indicate success or failure based on the response from the API.
+ 4. **Navigation**: A LinkButton is provided to navigate back to the home page 
+    after the acknowledgement is displayed.
+*/
+
+
+
+const GetActionMessage = (props) => {
+    const { t } = useTranslation();
+    if (props.isLoading) {
+      return t("SV_APPLICATION_PENDING");
+    }
+    return props.isSuccess  ? t("SV_SUBMIT_SUCCESSFULL") : t("SV_APPLICATION_FAILED");
+  };
+
+const rowContainerStyle = {
```

Comment:
Here as well which component it provides style

Reply:
comment added this style object will provide style to the banner which shows the application status  and application id